manual/python/index.md

```diff
@@ -81,31 +84,33 @@ ROOT.PyConfig.IgnoreCommandLineOptions = False
+If PyROOT cannot find `.rootlogon.py` in the user's home directory, it looks for the equivalent in C++ (`.rootlogon.C`), first in [ROOT's etc directory](https://root.cern/doc/master/classTROOT.html#ab8e51627a12d886d6c8177b46481352a){:target="_blank"}, then in the user's home directory and finally in the current working directory. 
 
-If you would like to completely disable the rootlogon functionality, set `PyConfig.DisableRootLogon` to `True`.
+Note that it is also possible to use both the Python and the C++ rootlogons, since the latter can be loaded from the former, for instance with `ROOT.gROOT.LoadMacro('.rootlogon.C')`.
 
-- `IgnoreCommandLineOptions` (default `True`): If a PyROOT script is executed with some command line arguments, they will be ignored by default by ROOT, so you can process them as you wish. However, by setting `PyConfig.IgnoreCommandLineOptions` to `False`, those arguments will be forwarded to ROOT for parsing, for example, to enable the batch mode from the command line. For a complete list of the arguments accepted by ROOT, → see [Starting ROOT with command line options]({{ '/manual/first_steps_with_root/#starting-root-with-command-line-options' | relative_url }}).
+If you want to disable the rootlogon functionality, set `PyConfig.DisableRootLogon` to `True`.
 
-- `ShutDown` (default `True`): when PyROOT is terminating, during its cleanup phase, the ROOT C++ interpreter is shut down. If PyROOT is executed as part of a longer-running application that needs the C++ interpreter, you can set `PyConfig.ShutDown` to `False` to prevent that shutdown.
+- `IgnoreCommandLineOptions` (default `True`): If a PyROOT script is run with some command line arguments, ROOT ignores them by default, so you can process them as you wish. However, by setting `PyConfig.IgnoreCommandLineOptions` to `False`, those arguments are forwarded to ROOT for parsing, for example, to enable the batch mode from the command line.<br/>For a complete list of the arguments accepted by ROOT, → see [Starting ROOT with command line options]({{ '/manual/first_steps_with_root/#starting-root-with-command-line-options' | relative_url }}).
 
-- `StartGUIThread` (default `True`): except when executed from [IPython](https://ipython.org/){:target="_blank"}, a [Jupyter](https://jupyter.org/){:target="_blank"} notebook or in [interactive mode](https://docs.python.org/3/tutorial/interpreter.html#interactive-mode){:target="_blank"}, PyROOT starts a thread that periodically polls for ROOT events (for example GUI events) to process them. If a given PyROOT application does not need this event processing, you can prevent the creation of the thread by setting `PyConfig.StartGUIThread` to `False`.
+- `ShutDown` (default `True`): If PyROOT is terminated during its cleanup phase, the ROOT C++ interpreter is shut down.<br/>If PyROOT is executed as part of a longer-running application that needs the C++ interpreter, you can set `PyConfig.ShutDown` to `False` to prevent that shutdown.
```

Comment:
I like it better how it was before, `If PyROOT is terminated during its cleanup phase` does not say what I meant. What about this: `When the application finishes, during PyROOT's cleanup phase, the ROOT C++ interpreter is shut down by default.`.

Reply:
Your proposal is indeed clearer. I think that the term "is terminated" is somehow more technical than "finishes", but may this is only a matter of taste.